auto_nag/scripts/spambug.py

```diff
@@ -0,0 +1,89 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import people
+from auto_nag.bugbug_utils import get_bug_ids_classification
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.utils import nice_round
+
+
+class SpamBug(BzCleaner):
+    def __init__(self):
+        super().__init__()
+        self.people = people.People.get_instance()
+
+    def description(self):
+        return "[Using ML] Detect spam bugs"
+
+    def columns(self):
+        return ["id", "summary", "confidence", "autofixed"]
+
+    def sort_columns(self):
+        return lambda p: (-p[2], -int(p[0]))
+
+    def handle_bug(self, bug, data):
+        reporter = bug["creator"]
+        if self.people.is_mozilla(reporter):
+            return None
+
+        return bug
+
+    def get_bz_params(self, date):
+        start_date, end_date = self.get_dates(date)
+
+        return {
+            "include_fields": ["id", "groups", "summary", "creator"],
+            # Ignore closed bugs.
+            "bug_status": "__open__",
+            "f1": "reporter",
+            "v1": "@mozilla",
+            "o1": "notsubstring",
+        }
```

Comment:
A couple more ideas to reduce the set of bugs, then we should be good to go.

- Skip bugs reported by Softvision (QA):
```
            "f2": "reporter",
            "v2": "@softvision",
            "o2": "notsubstring",
```

- Actually make use of start_date, considering only bugs which were recently opened:
```
            "f3": "creation_ts",
            "o3": "greaterthan",
            "v3": start_date,
```

Reply:
+1